cmd/node/config/economics.toml

```diff
@@ -40,6 +40,7 @@
     GasLimitSettings = [
         {EnableEpoch = 0, MaxGasLimitPerBlock = "1500000000", MaxGasLimitPerMiniBlock = "1500000000", MaxGasLimitPerMetaBlock = "15000000000", MaxGasLimitPerMetaMiniBlock = "15000000000", MinGasLimit = "50000"},
         {EnableEpoch = 2, MaxGasLimitPerBlock = "1500000000", MaxGasLimitPerMiniBlock = "600000000", MaxGasLimitPerMetaBlock = "15000000000", MaxGasLimitPerMetaMiniBlock = "600000000", MinGasLimit = "50000"},
+        {EnableEpoch = 3, MaxGasLimitPerBlock = "1500000000", MaxGasLimitPerMiniBlock = "500000000", MaxGasLimitPerMetaBlock = "15000000000", MaxGasLimitPerMetaMiniBlock = "500000000", MinGasLimit = "50000"},
```

Comment:
It seems that by mistake I have matched this epoch with the gas schedule v5 activation epoch:
```toml
[GasSchedule]
    GasScheduleByEpochs = [
        { StartEpoch = 0, FileName = "gasScheduleV1.toml" },
        { StartEpoch = 1, FileName = "gasScheduleV4.toml" },
        { StartEpoch = 3, FileName = "gasScheduleV5.toml" },
    ]
```